apollo-federation/src/query_plan/fetch_dependency_graph.rs

```diff
@@ -4125,29 +4134,34 @@ fn inputs_for_require(
     }
 }
 
+struct PostRequireInputsEdgeInfo<'a> {
```

Comment:
I'm not sure this grouping really makes things easier to understand. If it's just to appease clippy, it's okay to use an `#[allow()]` attribute on the function.